src/Core/Algorithms/Legacy/Inverse/TikhonovAlgoAbstractBase.cc

```diff
@@ -282,6 +290,9 @@ double TikhonovAlgoAbstractBase::computeLcurve( const SCIRun::Core::Algorithms::
 	// prealocate vector of lambdas and eta and rho
   std::vector<double> rho(nLambda, 0.0);
   std::vector<double> eta(nLambda, 0.0);
+  
+  lambdamatrix.reset(new DenseMatrix(nLambda,3,0.0));
+  double *data = lambdamatrix->data();
```

Comment:
It's preferred not to access matrix internals/pointers unless interfacing with C code or legacy modules. Please convert to using Eigen operator(,) syntax